### PR TITLE
Add a shortloglevel badips.py option

### PR DESCRIPTION
**Pull request by benrubson**
Hi,

This follows #2205 to fix the summary message loglevel.

Thank you 👍 

Ben

### CODE REVIEW

File: config/action.d/badips.py [hunk 1]
```diff
@@ -350,7 +350,7 @@ def update(self):
 				s = ips - self._bannedips
 				p = len(s)
 				self._banIPs(s)
-				self._logSys.log(self.loglevel,
+				self._logSys.info(
```
Comment (sebres) [inline config/action.d/badips.py]: Please change:
```diff
- logSys.log(loglevel,
+ logSys.log(logging.MSG,
```

**Comment by benrubson** [comment]
> I don't use badips, but cannot imagine for which reasons this should be logged after (successful) IPs-merge by the successful start of action (separated from other things, which are rather a debug purposes).

Think about merging sometimes hundreds of IPs.
You don't want an individual message for every processed IP, but rather a summary message with the number of removed/added IPs.
I agree that the _next update in X seconds_ part of the message is for sure debug.
What about then the commit I just pushed ? Would it be suitable ? We would not have "duplicate" messages.

> Anyway I can promise you, that tomorrow (as many times earlier) we'll get a request to disable this info, because it bothers someone (does not saw this message previously).

Sure, I understand.

**Comment by benrubson** [comment]
> So then why you have extended the logging in #2205 for "individual message for every processed IP" instead of doing this for summary message only?

The other reason is that you may want to have these individual messages for some of your badips jails, but not for some others (you can define several badips jails, so with different log levels).

> You still did not understand, I am not against this message, but against the **default** level (`INFO`) for it (for the reasons I already mentioned above).

Understood, let me then rework this correctly.

**Comment by benrubson** [comment]
I think this rework should be OK @sebres 👍 
We then have the `loglevel` option for the detailed (un)ban message.
And the `shortloglevel` option for the summary message.
We are then able to output both or only one of these 2 messages, independently, and at a jail level.

Thank you 👍

**Comment by benrubson** [comment]
@sebres what about merging this small PR ?
Many thanks again !

**Comment by benrubson** [comment]
Let's go with this if you're OK @sebres ?
Many thanks !

**Comment by benrubson** [comment]
Hi @sebres, may I ask you to review this tiny change please when you'll have some time ?
Many thanks !

**Comment by sebres** [comment]
It has been rewritten a bit: `loglevel` and `shortloglevel` combined now to single parameter `loglevel`.

Here an example logging summary with NOTICE and rest with DEBUG log-levels:
```ini
action = badips.py[loglevel="debug, notice"]
```
Parameter `shortloglevel` does not exists anymore.

**Comment by benrubson** [comment]
Really nice idea, many thx @sebres 👍

**Pull request merged**